src/v2/guide/migration.md

```diff
@@ -1257,57 +1257,65 @@ Los enlaces _one-time_ (`{% raw %}{{* foo }}{% endraw %}`) han sido reemplazados
 </div>
 {% endraw %}
 
-## Reactivity
+## Reactividad
 
-### `vm.$watch` <sup>changed</sup>
+### `vm.$watch` <sup>cambiado</sup>
 
-Watchers created via `vm.$watch` are now fired before the associated component rerenders. This gives you the chance to further update state before the component rerender, thus avoiding unnecessary updates. For example, you can watch a component prop and update the component's own data when the prop changes.
+Los _watchers_ creados a través de `vm.$watch` ahora se disparan antes de que el componente asociado se vuelva a renderizar. Esto le brinda la oportunidad de actualizar el estado antes de que el componente vuelva a renderizarse, evitando así actualizaciones innecesarias. Por ejemplo, puede escuchar por una propiedad y actualizar sus propios datos cuando la propiedad cambia.
 
-If you were previously relying on `vm.$watch` to do something with the DOM after a component updates, you can instead do so in the `updated` lifecycle hook.
+Si anteriormente usted dependía de `vm.$watch` para hacer algo con el DOM después de las actualizaciones de un componente, usted puede hacerlo en el enlace del ciclo de vida `updated`.
```

Comment:
I think we are actually leaving `_hook_` instead of translating it to `enlace`. We should discuss on what we are going to do, and maybe create a card on trello to check for all instances of `enlace/_hook_` and change accordingly.

Though on this?
cc: @sxcamacho

Reply:
Yes, I'm using _hook_ for example. I agree about create a card to update all occurrences.